### DISCUSSION TITLE
Mesh model freezing issue in imodeljs

### DISCUSSION THREAD

**Discussion by namsancow** [iTwin/itwinjs-core#169] [Q&A] [answered]
MicroStation has many element type such as surface, solid, cell, shape, and mesh etc. 

For surface, solid, cell, shape model, the published bim file works quickly in imodeljs. 
However for mesh model, the published bim file works very slowly. 
The imported mesh model from fbx, obj, 3ds file also works very slowly.

When bim file is published, the .Tiles and TileCache files are generated together. 
.Tiles and TileCache files used with .bim files in imodeljs when zooming and rotating.

For surface, solid, cell, shape model, .Tiles and TileCache files has no problem. 
However, for mesh model, the imodeljs is continually freezing. 

If the .TileCache file size is increased, can this freezing issue be solved?
Is there any method for increasing .TileCache file size?

**Comment by pmconne** [comment, marked as answer]
The .TileCache is not used by the iModel.js display system. It was probably created by the converter.
Are you able to supply the .dgn or .bim file?

**Reply by namsancow** [reply]
> The .TileCache is not used by the iModel.js display system. It was probably created by the converter.
> Are you able to supply the .dgn or .bim file?

Dear Paul
Many thanks for kind response.
Today I installed and tested the latest version of imodeljs and desktop-starter.
Unlike the old version, I found that the segmented tiles were uploaded very quickly, 
and that freezing of the model during rotation/movement had disappeared.
I think everything I've experienced in the old version has been solved in the latest version.
Thank you again for your kind advice.
